auditlog/__init__.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2013 ABF OSIELL (<http://osiell.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from . import models
+
+
+def pre_init_hook(cr):
+    cr.execute("SELECT 1 FROM pg_class WHERE relname = 'audittrail_rule'")
+    if cr.fetchall():
+        migrate_from_audittrail(cr)
+
+
+def migrate_from_audittrail(cr):
```

Comment:
Excellent to migrate existing audittrail rules. You should claim this feature in the README.rst.